app/src/main/res/layout/fragment_tool_details.xml

```diff
@@ -96,32 +96,31 @@
             android:text="@string/action_tools_remove"
             tools:visibility="visible" />
 
-        <org.ccci.gto.android.common.viewpager.view.ChildHeightAwareViewPager
-            android:id="@+id/detail_view_pager"
+        <com.google.android.material.tabs.TabLayout
+            android:id="@+id/detail_tab_layout"
             android:layout_width="match_parent"
             android:layout_height="wrap_content"
-            android:background="@color/gray_E6"
-            android:isScrollContainer="false">
+            android:elevation="7dp"
+            app:tabBackground="@color/action_bar_action_tint">
 
-            <com.google.android.material.tabs.TabLayout
-                android:id="@+id/detail_tab_layout"
-                android:layout_width="match_parent"
+            <com.google.android.material.tabs.TabItem
+                android:layout_width="wrap_content"
                 android:layout_height="wrap_content"
-                android:elevation="7dp"
-                app:tabBackground="@color/action_bar_action_tint">
+                tools:text="@string/label_tools_about" />
 
-                <com.google.android.material.tabs.TabItem
-                    android:layout_width="wrap_content"
-                    android:layout_height="wrap_content"
-                    tools:text="@string/label_tools_about" />
+            <com.google.android.material.tabs.TabItem
+                android:layout_width="wrap_content"
+                android:layout_height="wrap_content"
+                tools:text="40 Languages" />
```

Comment:
I don't think you actually need to have `TabItems` defined here. they should automatically be created when you link the TabLayout to the ViewPager